plugins/balloonpanel/plugin.js

```diff
@@ -320,10 +320,13 @@
 		 *
 		 * @method attach
 		 * @param {CKEDITOR.dom.element} element The element to which the panel is attached.
-		 * @param {CKEDITOR.dom.element/Boolean} [focusElement] The element to be focused after the panel
+		 * @param {Object} options Ballonpanel focus and show options
```

Comment:
This remains to be an optional parameter.

Reply:
Missing dot at the end.

Reply:
Typo "Ballonpanel" -> "Balloonpanel".

Reply:
"Balloonpanel focus and show options" - no need to enumerate option properties. If you do that you're putting yourself in responsibility of constantly maintaining this place (so that it is updated every time).

You don't want that, instead these options are listed automatically right below.